draft-ietf-quic-qpack.md

```diff
@@ -603,7 +607,8 @@ is used without modification.
 This document expands the definition of string literals and permits them to
 begin other than on a byte boundary.  An "N-bit prefix string literal" begins
 with the same Huffman flag, followed by the length encoded as an (N-1)-bit
-prefix integer.  The remainder of the string literal is unmodified.
+prefix integer.  The prefix size, N, can have a value between 2 and 8.  The
```

Comment:
I didn't see a 2-bit prefix anywhere, should this be 3 and 8 (inclusive)?

But I see this as a bit of overkill.  If we allow 1 or 2 in QPACK2 we have to remember to change this.

Reply:
For comparison, RFC7541 5.1 specificly allows the prefix length to be between 1 and 8, inclusive, even though prefix lengths 1, 2, and 8 are never used in HPACK.  And that's great because now QPACK does not have to call out that the prefix can be 8 bits (for example, for Encoder Required Insert Count).  So I suggest to be as permissive here as reasonable.

If we allow N to be 1, then after the Huffman bit, the length is encoded as an 8-bit prefix integer, and not an N-1 = 0 bit prefix integer.  Is this worth adding here given that it's not used?

Reply:
Ah, right, N=1 doesn't make sense.  2 and 8 inclusive?

Reply:
Okay.  I guess it is the word "inclusive" that you are missing, so I'm adding it.

It is interesting (but irrelevant) to note that RFC7541 5.1 does not explicitly include the word "inclusive", though the next sentence implies that 8 is allowed:
"The prefix size, N, is always between 1 and 8 bits. An integer starting at an octet boundary will have an 8-bit prefix."